app/default/content_routes.py

```diff
@@ -13,10 +15,12 @@ def accessibility_statement():
     return render_template("accessibility_statement.html")
 
 
-@content_bp.route("/cof_r2w2_all_questions", methods=["GET"])
-def all_questions():
-    current_app.logger.info("All questions page loaded.")
-    return render_template("cof_r2w2_all_questions.html")
+@content_bp.route("/all_questions/<application_id>", methods=["GET", "POST"])
+def all_questions(application_id):
+    application = get_application_data(application_id, as_dict=True)
+    with force_locale(application.language):
```

Comment:
Perhaps it would be cleaner to allow a `lang` param on the language switcher e.g. in https://github.com/communitiesuk/funding-service-design-utils/blob/bafc22f12d359ccad3dc62ee8830d68bba279e8c/fsd_utils/locale_selector/get_lang.py#L6 rather than changing the route like this? Or if they are already on a welsh application, will they not already have the cookie set to pick up the welsh version?

Reply:
I enforced it because the functionality was suggested by you when we were working on the welsh translation. Happy to discuss and also dropped a text to @srh-sloan

Reply:
As discussed, I am closing this PR and try to fix the utils instead.